docs/getting-started.md

```diff
@@ -70,7 +70,7 @@ dockerizied environment, clone the KubeVirt repository, `cd` into it, and:
 
 ```bash
 # Build and deploy KubeVirt on Kubernetes in our vms inside containers
-export KUBEVIRT_PROVIDER=k8s-1.13.3 # this is also the default if no KUBEVIRT_PROVIDER is set
+export KUBEVIRT_PROVIDER=k8s-1.15 # this is also the default if no KUBEVIRT_PROVIDER is set
```

Comment:
this is going to constantly be out of date as we move forward. any ideas on how we can make this accurate without having to update this document every time our ci providers change?

Reply:
Yes. We have some releng tooling in project-infra which can determine the latest version and push a file with the tag as content to gcs. It can then be downloaded via CURL.

We still need to create a job which does that.